packages/core/test/client.test.ts

```diff
@@ -36,6 +36,37 @@ describe('@bugsnag/core/client', () => {
         unhandledRejections: true
       })
     })
+
+    it('warns with a valid but incorrect-looking api key', () => {
```

Comment:
Do _valid but incorrect-looking_ api keys exist?

Reply:
Valid in the sense that the notifier will accept it, but not one our app would ever generate. This allows us to put a placeholder API key in examples/snippets etc. that will still run properly, but not send errors to any dashboard.